backend/codesirius/codesirius/exception_handler.py

```
@@ -39,6 +58,8 @@ def build_response(response, status_code, message, path, errors=None):
 
 
 def custom_exception_handler(exc, context):
+    logger.error(f"An exception occurred: {exc}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)